library/common/2206.0.0/templates/lib/storage/_hostpathValidation.tpl

```diff
@@ -0,0 +1,56 @@
+{{/*
+Validate list of host path in a specific format
+*/}}
+{{- define "common.storage.hostPathsValidation" -}}
+    {{- $allowed_paths := (list "mnt" "sys" "dev" "cluster") -}}
+    {{- range . -}}
+        {{- $host_p := splitList "/" . -}}
+        {{- $host_p := (without $host_p "") -}}
+        {{- $error_msg := (printf "Invalid hostpath %s. Path must be a valid path under a given pool e.g `/mnt/tank/somepath` is valid whereas `/mnt` or `/mnt/tank` are invalid examples." .) -}}
```

Comment:
What is the purpose of this validation?

The message doesn't explain itself, nor does the commit message. Inability to mount `/mnt/tank` makes maintaining the configuration rather awkward.